src/client/stdlib/duckdb.js

```diff
@@ -29,16 +29,26 @@ import * as duckdb from "npm:@duckdb/duckdb-wasm";
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-const bundle = await duckdb.selectBundle({
-  mvp: {
-    mainModule: import.meta.resolve("npm:@duckdb/duckdb-wasm/dist/duckdb-mvp.wasm"),
-    mainWorker: import.meta.resolve("npm:@duckdb/duckdb-wasm/dist/duckdb-browser-mvp.worker.js")
-  },
-  eh: {
-    mainModule: import.meta.resolve("npm:@duckdb/duckdb-wasm/dist/duckdb-eh.wasm"),
-    mainWorker: import.meta.resolve("npm:@duckdb/duckdb-wasm/dist/duckdb-browser-eh.worker.js")
-  }
-});
+// Baked-in manifest.
+const manifest = process.DUCKDB_MANIFEST;
```

Comment:
Thinking out loud… baking the manifest into /_observablehq/stdlib/duckdb.js means that the content hash will change when the DuckDB manifest changes… and that by extension the module hash of /_observablehq/stdlib.js will change because it imports /_observablehq/stdlib/duckdb.js. We should double-check that this works but I believe the hashing is fairly robust now. 😅

Reply:
I think this works, yes. But always double check.